src/Stick.js

```diff
@@ -73,6 +73,15 @@ function Stick({
     })
   )
 
+  /* Check alignment on first render */
+  useEffect(() => {
+    if (!nodeRef.current || !anchorRef.current) {
+      return
+    }
+
+    checkAlignment(nodeRef.current, anchorRef.current)
+  }, [nodeRef.current, anchorRef.current])
+
```

Comment:
With the given dependency array this effect would potentially use an outdated handler. However, when I look at this I think the same can be achieved by *calling* `checkAlignment` in the effect underneath this one. There we register the `scroll` handler. But we could also initially call it once to make sure the alignment is correct *right now*.